test/trello-provider.test.ts

```diff
@@ -0,0 +1,146 @@
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
+
+import * as Fs from 'fs'
+
+import TrelloProvider from '../src/trello-provider'
+
+
+const Seneca = require('seneca')
+const SenecaMsgTest = require('seneca-msg-test')
+const TrelloProviderMessages = require('./trello-provider.messages').default
+
+const CONFIG: any = {}
+
+if (Fs.existsSync(__dirname + '/local-config.js')) {
+    Object.assign(CONFIG, require(__dirname + '/local-config.js'))
+}
+
+jest.setTimeout(10000)
+
+describe('trello-provider', () => {
+
+    test('happy', async () => {
+        const seneca = Seneca({legacy: false})
```

Comment:
this test should still pass in Github Actions however as it just loads the plugin and does not not make calls needing the api keys

Reply:
Precisely. So, in this case, I passed the "missingKeys" value, as a property of our test suite provider. As the following:

```js
test: {
    value: missingKeys
  }
```

Allowing us to validate its value, in our provider (seneca.prepare) and load the plugin without the credentials. 

However, if you think is better that we use another solution, just let me know.